### PR TITLE
Proposing Bose, Raman and Ramanujan

### PR DESCRIPTION
**Pull request by sanketsaurav**
Proposing to add notable Indian scientists Satyendra Nath Bose, C. V. Raman and Srinivasa Ramanujan to the random names generator.

Signed-off-by: Sanket Saurav sanketsaurav@gmail.com

### CODE REVIEW

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -107,6 +107,9 @@ var (
 
 		// Niels Bohr is the father of quantum theory. https://en.wikipedia.org/wiki/Niels_Bohr.
 		"bohr",
+		
+		// Satyendra Nath Bose - He provided the foundation for Bose–Einstein statistics and the theory of the Bose–Einstein condensate. - http://en.wikipedia.org/wiki/Satyendra_Nath_Bose
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Could you change the `http://` links to `https://` (here, and below)?

**Comment by thaJeztah** [comment]
@sanketsaurav you forgot to sign-off your commits. You can fix this using the instructions in https://github.com/docker/docker/pull/13597#issuecomment-106893923

LGTM, after signing off and addressing my comment above.

**Comment by sanketsaurav** [comment]
@thaJeztah Done, and done!

**Comment by estesp** [comment]
Looks good.. would you mind just merging the two commits into one signed commit, and then we can merge? Thanks!

**Comment by sanketsaurav** [comment]
@estesp Done. Are we good to go now?

**Comment by estesp** [comment]
yes, we just have to wait for CI to complete.  Thanks!

LGTM after CI

**Comment by thaJeztah** [comment]
code and description LGTM (if we agree these names should be added)

**Comment by thaJeztah** [comment]
ping @cpuguy83 for a final LGTM (and an update to `docker-who`) 😉

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**